css/style.css

```diff
@@ -0,0 +1,918 @@
+* {
+  box-sizing: border-box;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  background-color: #eee;
+  z-index: 2;
+}
+
+.tablet,
+.nav1,
+.nav2 {
+  display: none;
+}
+
+.align {
+  align-items: center;
+  justify-content: center;
+}
+
+.tnw-logo {
+  display: flex;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  width: 100px;
+}
+
+.tnw-logo img {
+  width: 76px;
+  max-height: 28px;
+}
+
+.right {
+  display: flex;
+  max-width: 100px;
+  border: 1px solid rgb(194, 177, 177);
+  text-align: center;
+}
+
+.search-img,
+.menu-icon {
+  width: 48px;
+  height: 48px;
+  display: flex;
+}
+
+.search-img img,
+.menu-icon img {
+  width: 20px;
+  height: 20px;
+}
+
+.search-img {
+  border-right: 1px solid gainsboro;
+}
+
+.nav3 {
+  width: 100%;
+  height: 54px;
+  background-color: yellow;
+  display: flex;
+  font-size: 14px;
+  position: fixed;
+  margin-top: 50px;
+  align-items: center;
+  justify-content: center;
+  z-index: 3;
+}
+
+.vid {
+  margin-top: 90px;
+  width: 100%;
+}
+
+.art1,
+.art2,
+.art3 {
+  position: relative;
+  width: 96%;
+  height: 180px;
+  margin-left: 10px;
+  margin-top: 5px;
+}
+
+.art1 {
+  background: linear-gradient(180deg, rgba(177, 80, 85, 0.7) 0%, rgba(0, 0, 0, 0.9) 100%), url(../ressources/article-img.jpg);
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.art2 {
+  background: linear-gradient(180deg, rgba(255, 31, 42, 0.7) 0%, rgba(0, 0, 0, 0.9) 100%), url(../ressources/article-image2.jpg);
+}
+
+.art3 {
+  background: linear-gradient(180deg, rgba(20, 35, 243, 0.7) 0%, rgba(0, 0, 0, 0.9) 100%), url(../ressources/article-image3.jpg);
+}
+
+.alt-img {
+  display: none;
+}
+
+.art-contents {
+  position: absolute;
+  bottom: 2rem;
+  margin-left: 0.5em;
+}
+
+.art-contents h2 {
+  font-size: 1.5em;
+  color: #ddd;
+  text-align: justify;
+  line-height: 1em;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.art-foot {
+  display: inline-flex;
+  color: #ddd;
+}
+
+.art-foot li {
+  list-style-type: none;
+  font-size: 0.9em;
+}
+
+.grid-img img {
+  width: 80px;
+  height: 80px;
+}
+
+.paragraph img {
+  width: 31px;
+  height: auto;
+}
+
+.starz img {
+  width: 35px;
+}
+
+.app-img img {
+  width: 95%;
+}
+
+.right-img img {
+  width: 100px;
+  height: 90px;
+}
+
+.art-foot li img {
+  width: 12px;
+  height: 10px;
+}
+
+.bottom-context p {
+  text-align: center;
+}
+
+.latest-news a {
+  display: flex;
+  text-decoration: none;
+  color: red;
+  align-items: center;
+  justify-content: space-between;
+  width: 95%;
+}
+
+.fund-cap,
+.funding-bottom a {
+  text-decoration: none;
+}
```

Comment:
Same styles of the selector as the line 274. In order to minimize code, you can group selectors with the same style definition.